cognite/neat/rules/exporters/_rules2excel.py

```diff
@@ -194,3 +211,74 @@ def _adjust_column_widths(cls, workbook: Workbook) -> None:
                 current = sheet.column_dimensions[selected_column.column_letter].width or (max_length + 0.5)
                 sheet.column_dimensions[selected_column.column_letter].width = max(current, max_length + 0.5)
         return None
+
+    def _create_metadata_sheet_user_rules(self, rules: Rules) -> dict[str, Any]:
+        metadata: dict[str, Any] = {
+            field_alias: None for field_alias in rules.metadata.model_dump(by_alias=True).keys()
+        }
+        if "creator" in metadata:
+            metadata["creator"] = "YOUR NAME"
+
+        if isinstance(rules, DomainRules):
+            return metadata
+        elif isinstance(rules, DMSRules):
+            existing_model_id = (rules.metadata.space, rules.metadata.external_id, rules.metadata.version)
+        elif isinstance(rules, InformationRules):
+            existing_model_id = (rules.metadata.prefix, rules.metadata.name, rules.metadata.version)
+        else:
+            raise ValueError(f"Unsupported rules type: {type(rules)}")
+        existing_metadata = rules.metadata.model_dump(by_alias=True)
+        if isinstance(existing_metadata["created"], datetime):
+            metadata["created"] = existing_metadata["created"].replace(tzinfo=None)
+        if isinstance(existing_metadata["updated"], datetime):
+            metadata["updated"] = existing_metadata["updated"].replace(tzinfo=None)
+        # Excel does not support timezone in datetime strings
+        now_iso = datetime.now().replace(tzinfo=None).isoformat()
+        is_info = isinstance(rules, InformationRules)
+        is_dms = not is_info
+        is_extension = self.new_model_id is not None
+        is_solution = is_extension and self.new_model_id != existing_model_id
+
+        if is_solution:
+            metadata["prefix" if is_info else "space"] = self.new_model_id[0]  # type: ignore[index]
+            metadata["title" if is_info else "externalId"] = self.new_model_id[1]  # type: ignore[index]
+            metadata["version"] = self.new_model_id[2]  # type: ignore[index]
+        else:
+            metadata["prefix" if is_info else "space"] = existing_model_id[0]
+            metadata["title" if is_info else "externalId"] = existing_model_id[1]
+            metadata["version"] = existing_model_id[2]
+
+        if is_solution and is_info:
+            metadata["namespace"] = f"http://purl.org/{self.new_model_id[0]}/{self.new_model_id[1]}"  # type: ignore[index]
```

Comment:
do not add title ! as title can be any set of strings, just use prefix, ie:
f"http://purl.org/{self.new_model_id[0]}/"